tests/rptest/tests/cluster_self_config_test.py

```diff
@@ -85,6 +85,8 @@ def test_s3_self_config(self, cloud_storage_type):
 
             assert self_config_result and self_config_result in self_config_expected_results
 
+    # OCI only supports path-style requests, fips mode will always fail.
+    @ok_to_fail_fips
```

Comment:
nit: should we introduce @skip_fips instead so we don't waste CI time with them? See `skip_debug_mode` definition

Reply:
https://github.com/redpanda-data/redpanda/pull/23331 Done